src/import/importd.c

```diff
@@ -1125,131 +1126,114 @@ static int property_get_progress(
-                                 SD_BUS_PARAM(format),
-                                 "uo",
-                                 SD_BUS_PARAM(transfer_id)
-                                 SD_BUS_PARAM(transfer_path),
-                                 method_export_tar_or_raw,
-                                 SD_BUS_VTABLE_UNPRIVILEGED),
-        SD_BUS_METHOD_WITH_NAMES("PullTar",
-                                 "sssb",
-                                 SD_BUS_PARAM(url)
-                                 SD_BUS_PARAM(local_name)
-                                 SD_BUS_PARAM(verify_mode)
-                                 SD_BUS_PARAM(force),
-                                 "uo",
-                                 SD_BUS_PARAM(transfer_id)
-                                 SD_BUS_PARAM(transfer_path),
-                                 method_pull_tar_or_raw,
-                                 SD_BUS_VTABLE_UNPRIVILEGED),
-        SD_BUS_METHOD_WITH_NAMES("PullRaw",
-                                 "sssb",
-                                 SD_BUS_PARAM(url)
-                                 SD_BUS_PARAM(local_name)
-                                 SD_BUS_PARAM(verify_mode)
-                                 SD_BUS_PARAM(force),
-                                 "uo",
-                                 SD_BUS_PARAM(transfer_id)
-                                 SD_BUS_PARAM(transfer_path),
-                                 method_pull_tar_or_raw,
-                                 SD_BUS_VTABLE_UNPRIVILEGED),
-        SD_BUS_METHOD_WITH_NAMES("ListTransfers",
-                                 NULL,,
-                                 "a(usssdo)",
-                                 SD_BUS_PARAM(transfers),
-                                 method_list_transfers,
-                                 SD_BUS_VTABLE_UNPRIVILEGED),
-        SD_BUS_METHOD_WITH_NAMES("CancelTransfer",
-                                 "u",
-                                 SD_BUS_PARAM(transfer_id),
-                                 NULL,,
-                                 method_cancel_transfer,
-                                 SD_BUS_VTABLE_UNPRIVILEGED),
-
-        SD_BUS_SIGNAL_WITH_NAMES("TransferNew",
-                                 "uo",
-                                 SD_BUS_PARAM(transfer_id)
-                                 SD_BUS_PARAM(transfer_path),
-                                 0),
-        SD_BUS_SIGNAL_WITH_NAMES("TransferRemoved",
-                                 "uos",
-                                 SD_BUS_PARAM(transfer_id)
-                                 SD_BUS_PARAM(transfer_path)
-                                 SD_BUS_PARAM(result),
-                                 0),
-
-        SD_BUS_VTABLE_END,
+static const bus_object_implementation manager_object = {
+        "/org/freedesktop/import1",
+        "org.freedesktop.import1.Manager",
+
+        .vtable =
+        {SD_BUS_VTABLE_START(0),
```

Comment:
I wanted to minimize the indentation. If a newline is used, everything get's pushed to the right...

Reply:
well, but now you are pushing it one character to the right... I'd just place the bracket after the `=` and then manually remove the indentation emacs wants to add on the next line. it will then follow through with the same indentation for all further lines